Shared/Samples/Set up location-driven geotriggers/SetUpLocationDrivenGeotriggersView.swift

```diff
@@ -0,0 +1,336 @@
+            portal: .arcGISOnline(connection: .anonymous),
+            id: .santaBarbaraBotanicGardenMap
+        ))
+        
+        /// The location display for the map view.
+        lazy var locationDisplay = makeLocationDisplay()
+        
+        /// The route simulated location data source.
+        private let locationDataSource: SimulatedLocationDataSource
+        
+        /// An array of geotrigger monitors.
+        private var geotriggerMonitors: [GeotriggerMonitor] = []
+        
+        /// The name of the current garden section feature.
+        var currentSectionName: String? {
+            featureNamesInFenceGeotrigger[sectionFenceGeotriggerName]?.last
+        }
+        
+        /// The names of nearby point-of-interest features.
+        var nearbyPOINames: [String] {
+            featureNamesInFenceGeotrigger[poiFenceGeotriggerName] ?? []
+        }
+        
+        /// A dictionary for nearby features.
+        var nearbyFeatures: [String: ArcGISFeature] = [:]
+        
+        /// A dictionary for the feature names in each fence geotrigger.
+        /// - Key: The name of a fence geotrigger.
+        /// - Value: An array of names of features within the fence.
+        private var featureNamesInFenceGeotrigger: [String: [String]] = [:] {
+            didSet {
+                hasCurrentSection = currentSectionName != nil
+                hasPointOfInterest = !nearbyPOINames.isEmpty
+            }
+        }
+        
+        /// A string for the fence geotrigger notification status.
+        @Published var fenceGeotriggerText = "Fence geotrigger info will be shown here."
+        
+        /// A string for the display name of the currently nearby feature.
+        @Published var nearbyFeaturesText = "Nearby features will be shown here."
+        
+        /// A Boolean value indicating whether there is a current section.
+        @Published var hasCurrentSection = false
+        
+        /// A Boolean value indicating whether there is a point-of-interest.
+        @Published var hasPointOfInterest = false
+        
+        /// A Boolean value indicating whether to show an error alert.
+        @Published var isShowingAlert = false
+        
+        /// The error shown in the alert.
+        @Published var error: Error? {
+            didSet { isShowingAlert = error != nil }
+        }
+        
+        /// The handle for the geotrigger monitor notification task.
+        private var notificationTaskHandle: Task<Void, Never>!
+        
+        init() {
+            // Make data source
+            // Densify the polyline to control the simulation speed.
+            let routePolyline = try? Polyline.fromJSON(walkingTourPolylineJSON)
+            let densifiedRoute = GeometryEngine.geodeticDensify(
+                routePolyline!,
+                maxSegmentLength: 5.0,
+                lengthUnit: .meters,
+                curveType: .geodesic
+            ) as! Polyline
```

Comment:
Since the end result we want is a polyline, put it on the `Polyline` extension at the bottom to declutter the model.

```swift
private extension Polyline {
    /// A path in Santa Barbara Botanic Garden.
    static var walkingTour: Polyline {
        let walkingTourPolylineJSON = ...
        let routePolyline = try? Polyline.fromJSON(walkingTourPolylineJSON)
        let densifiedRoute = GeometryEngine.geodeticDensify(
            routePolyline!,
            maxSegmentLength: 5.0,
            lengthUnit: .meters,
            curveType: .geodesic
        ) as! Polyline
        return densifiedRoute
    }
}
```